### PR TITLE
[15.0][MIG] purchase_batch_invoicing: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
cc @Tecnativa TT36502

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@pedrobaeza please review

**Comment by cesar-tecnativa** [comment]
@ernestotejeda please review

**Comment by cesar-tecnativa** [comment]
@sergio-teruel could you please review?

File: purchase_batch_invoicing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Batch Invoicing",
+    "summary": "Make invoices for all ready purchase orders",
+    "version": "15.0.1.1.0",
```
Comment (ernestotejeda) [inline purchase_batch_invoicing/__manifest__.py:6]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
This module is not needed since version 14...
Sorry!! but before migrate a module we have to ensure that the behaviour of the developed module is still mandatory...

**Pull request closed without merging**